build.gradle

```diff
@@ -1361,6 +1375,21 @@ project(':clients') {
     generator project(':generator')
   }
 
+  shadowJar {
+    archiveClassifier = null
```

Comment:
can we add a comment to explain why archiveClassifier is set to null?

Reply:
Done.